corehq/__init__.py

```diff
@@ -1,3 +1,4 @@
 from .celery import app as celery_app  # noqa
 
+# This long is intentionally too long. This long is intentionally too long. This long is intentionally too long. This long is intentionally too long.
```

Comment:
This is the commit https://github.com/dimagi/commcare-hq/commit/35ccc8baa24832591481646f3eeecebf937e0794 (which I've since removed from the PR) that this stickler comment was in response to.

Reply:
To me this shows that stickler is getting it's config from the right place (the `.flake8` file), because it's has a max line length of 115 which isn't the default but is what we have in `.flake8` (and nowhere else).